casanovo/casanovo.py

```
@@ -181,9 +181,7 @@ def sequence(
     utils.log_system_info()
 
     utils.check_dir_file_exists(output_path, f"{output_root}.mztab")
-    config, model = setup_model(
-        model, config, output_path, output_root_name, False
-    )
+    config, model = setup_model(model, config, output_path, output_root_name, False)
```

Ruff (0.15.7)

[warning] Boolean positional value in function call

(FBT003)